scripts/jenkins/init-node.sh

```diff
@@ -8,7 +8,7 @@ nvm install
 
 # clean out and install yarn
 rm -rf ~/.yarn
-curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 0.21.3
+curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.3.2
```

Comment:
We should have such values in one place, so we don't have to update them in several files each time. E.g. have a `scripts/env.sh` and import it in all places where we need them.

Reply:
We can't cover all of them as we can't import anything in `package.json`. But Jenkins/Travis versions could be shared, yes.